next.config.js

```diff
@@ -17,6 +17,7 @@ module.exports = withBundleAnalyzer({
         ANALYSIS_SUPPORT_USER: config.get("analysis.support.user"),
         ANALYSIS_SUPPORT_SOURCE_ID: config.get("analysis.support.source_id"),
         IRODS_HOME_PATH: config.get("irods.home_path"),
+        IRODS_TRASH_PATH: config.get("irods.trash_path"),
```

Comment:
This trash path config value was already made available server side, but I never added it client side, so this includes those changes.